notebooks/D1-entitlement-demographics.ipynb

```diff
@@ -15,73 +15,148 @@
    "source": [
     "import intake\n",
     "import IPython.display\n",
+    "import ipywidgets\n",
     "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
     "import pandas\n",
+    "import pcts_census_utils\n",
     "\n",
     "cat = intake.open_catalog(\"../catalogs/*.yml\")"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 2,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "# Load PCTS extract\n",
-    "pcts = cat.pcts2.read().reset_index(drop=True)"
```

Comment:
This `reset_index` actually winds up being pretty important, and was a tricky bug to track down. I want to make sure we don't revert the fix.

Reply:
I don't really understand how the reset_index and the `subset_pcts` functions work together. Does it matter? I know it matters if you're using a regex directly, but I want to move into a uniform way of getting our PCTS data, whether it's for regression or for TOC analysis (theoretically, TOC is only a subset of the same dataset used for regression).

Reply:
I agree with you that we should have a uniform way. The problem I ran into was that the master pcts file actually had a messed up index in s3, which was messing with some of the joins. It was pretty subtle and difficult to catch, so I just want to make sure that `subset_pcts` doesn't have the same problem.